README.md

```diff
@@ -1,2 +1,68 @@
 # django-ilmoitin
-Django app for sending notifications
+
+[![Latest PyPI version](https://badge.fury.io/py/django-ilmoitin.svg)](https://pypi.python.org/pypi/django-ilmoitin)
+[![Python versions](https://img.shields.io/pypi/pyversions/django-ilmoitin.svg)](https://pypi.python.org/pypi/django-ilmoitin)
+
+A templated Django messaging library
+
+# Installation
+
+1. `pip install django-ilmoitin`
+
+2. Add `django_ilmoitin` to `INSTALLED_APPS`.
+
+# Usage
+
+1. Django ilmoitin uses Django Mailer to send emails, so you need to configure the MAILER_EMAIL_BACKEND setting, that django-mailer knows how to actually send the mail
```

Comment:
I would highlight MAILER_EMAIL_BACKEND with '`' : `MAILER_EMAIL_BACKEND`

Reply:
as well as use `django-ilmoitin` and `django-mailer` everywhere, and probably link `django-mailer` to their repo https://github.com/pinax/django-mailer